utils/build/docker/set-system-tests-weblog-env.Dockerfile

```diff
@@ -14,6 +14,12 @@ ENV DD_TRACE_COMPUTE_STATS=true
 
 ENV SOME_SECRET_ENV=leaked-env-var
 
+# Special setup for DD_DATA_STREAMS_ENABLED
+# until we figure out the issue with enabling
+# data streams on spring-boot-native
+ARG DD_DATA_STREAMS_ENABLED
```

Comment:
See comment above (move those line in `spring-boot-native.Dockerfile`

Reply:
done. I actually only turned on DD_DATA_STREAMS_ENABLED in the dockerfiles where we are testing the configs